opentracing-kotlin/README.md

```diff
@@ -0,0 +1,79 @@
+# Opentracing Kotlin
+====================
+
+This module has been designed to help Kotlin/Java developers reduce boiler plate code while intrumenting opentracing in
```

Comment:
have you tried if the methods feel natural to use in Java? If not, I'd remove `Java` here to avoid creating expectations.

Reply:
I see that you have it in the docs - the extension methods translate to static method calls in Java, which is OK, but not the most idiomatic - I guess I'd use a wrapper - e.g. "RichTracer", which has those methods.
In other words, I would create something dedicated for Java instead - or just scope it out from the docs.

Reply:
Agreed that the java concept is not ideal, but until something like that is available, this could be an alternative. In fact, we do use this for one of our java projects. But, sure I can remove it from the doc.